.github/workflows/test-rails.yaml

```
@@ -60,6 +60,11 @@ on:
         description: 'Password for the MySQL database'
         required: false
         type: string
+      extraSystemDependencies:
+        description: 'Install additional system dependenices'
```

typo (similarly below on L87)
```suggestion
        description: 'Install additional system dependencies'
```